modules/apps/27-interchain-accounts/host/types/keys.go

```diff
@@ -4,6 +4,9 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+// ParamsKey is the key to use for the storing params.
+var ParamsKey = []byte{0x00}
```

Comment:
I wonder if we should just use a string value here? Not a big deal either way:

pros:
- byte savings (5-10 bytes), but key is only written once

cons:
- harder to maintain, not possible to assert another key doesn't use the same value
- not a const

Our codebase is a bit split, but the majority of the time we using strings